### DISCUSSION TITLE
Export results incomplete

### DISCUSSION THREAD

**Discussion by stthi** [asreview/asreview#1630] [Q&A] [answered]
Hello ASreview community,

I am using ASreview and I am now trying to export my results. However, the export does not show all the inclusion/exclusion decisions I made. I have an export of all (>17000 articles) I have looked at. This excel file shows only 917 articles with a 0 or 1 in the 'included' column, which should be almost 4000. How can I fix this?

Thanks!

**Comment by Rensvandeschoot** [comment]
So, are all papers that are unseen not part of the export? The export file should contain all records (see the [docs](https://asreview.readthedocs.io/en/latest/progress.html#export-results)), but not in your case. If you export your data as CSV file, do you get all the papers?

**Reply by stthi** [reply]
The export contains all records, however it does not show all inclusion decisions.

**Reply by Rensvandeschoot** [reply]
What is shown on the [analytics](https://asreview.readthedocs.io/en/latest/progress.html#analytics) page?

**Comment by stthi** [comment]
[image: image.png]
This is my analytics page

Op do 4 jan 2024 om 16:28 schreef Rens van de schoot <
***@***.***>:

**Reply by Rensvandeschoot** [reply]
Can you upload the file?

**Reply by stthi** [reply]
![analytics](https://github.com/asreview/asreview/assets/155644917/51a470bd-0c8b-4543-8f6c-0d419688e5d3)

**Reply by Rensvandeschoot** [reply, marked as answer]
This means there are 17.406 records in your dataset and 3,976 records have been screened of which 706 are labeled relevant. If you export the data as EXCEL (or CSV) the data is ordered in a specific way: It should be the case that the first 706 rows are the relevant records in the order they were shown during the screening process. Then, the next 13.430 rows are the records not seen during the screening, which are ordered from most to least relevant according to the last iteration of the model. The final 3270 rows in the file should be the non-relevant records in the order these are shown during the screening process.

**Reply by stthi** [reply]
Oh I see it now, thank you
